test/pageobjects/homePage.js

```diff
@@ -0,0 +1,29 @@
+const BasePage = require(`./basePage.js`);
+const Button = require(`../pageobjects/commonWebComponents/components/button.js`);
+
+class HomePage extends BasePage {
+
+   
+    get loginButton() {return new Button('[ng-reflect-router-link="/login"]')}
+    get registerButton() {return new Button('[ng-reflect-router-link="/register"]')}
+    get bidNowButton() {return new Button('[class="highlight__btn"]')}
```

Comment:
You can make new "selectors.js" utility class to store all your element selectors.
This can help centralise and organise your selectors in one place, making it easier to manage and update them.